lib/3scale/backend/stats/cleaner.rb

```diff
@@ -197,6 +221,30 @@ def service_from_stats_key(stats_key)
             # simply ignore those keys.
             nil
           end
+
+          def delete_stats_keys_with_val_0(redis_conn, log_deleted_keys)
+            cursor = 0
+
+            loop do
+              cursor, keys = redis_conn.scan(cursor, count: SCAN_SLICE)
+
+              stats_keys = keys.select { |k| is_stats_key?(k) }
+
+              unless stats_keys.empty?
+                values = redis_conn.mget(*(stats_keys.to_a))
```

Comment:
Isn't `.to_a` superfluous because `stats_keys` is just a simple array with the keys? I think this would be `redis_conn.mget(*stats_keys)`.

Reply:
:man_facepalming:

Reply:
Fixed.